recipes/anyio/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "anyio" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "0f47b5ea774cc9d403935c617ff27c7673f87cda0688cd1aab3137ac86283a68"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools_scm
+  run:
+    - python >=3.6
```

It seems that there are some dependencies for this project
```suggestion
    - python >=3.6
    - sniffio
    - curio
    - trio
    
```